scripts/gen_release_notes/generate.py

```diff
@@ -0,0 +1,81 @@
+"""
+This is a python approach to the problem of doing the following:
+- Grab all of the file changes
+- Grab the difference between two points of time
+- Generate a markdown file that contains this automated change log
+
+The goal of this is to make this as universal as possible, but also keep it Python native
+"""
+
+import argparse
+import json
+import os
+import sys
+
+from git import Repo
+
+from file_writer import file_writer
+from git_parser import git_parser
+
+# Global Configuration
+try:
+    custom_path = os.path.expanduser("~") + "/.config/.gen_release.config"
+    file_path = os.getenv("CONFIG_LOC", custom_path)
+    with open(file_path, "r") as config_file:
+        json_obj = json.load(config_file)
+except FileNotFoundError:
+    print("Error, there doesn't exist a configuration file at {path}".format(path=file_path))
+    print("Place a config file in said path or specify a custom path via setting CONFIG_LOC.")
+    print("A sample config can be obtained in Nebulaworks/orion/scripts/gen_release_notes/gen_release_sample.config")
+    sys.exit(1)
+
+main_branch_name = json_obj["main_branch_name"]
+dev_env_name = json_obj["dev_env_name"]
+tst_env_name = json_obj["tst_env_name"]
```

Comment:
can we go call this `stg` for `staging` instead? Only reason I say that is calling this `tst` is somewhat confusing and misleading.

Reply:
Oh good catch on that, I can refactor this script to use that name instead.